glusterd2/commands/snapshot/snapshot-create.go

```diff
@@ -568,6 +568,7 @@ func createSnapinfo(c transaction.TxnCtx) error {
 	}
 
 	snapVolinfo.State = volume.VolCreated
+	snapVolinfo.GraphMap = make(map[string]string)
```

Comment:
This will be over-written below. `volinfo.GraphMap` may also be nil

Reply:
same here

Reply:
As per your comment, the values will be over-written (=) only by non-nil values, in which case there's no need for this change at all. Isn't it ?